src/main/java/seedu/address/model/task/Location.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.task;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.contact.Name;
+
+public class Location {
+    public final String location;
+
+    public Location(String location) {
+        requireNonNull(location);
+        this.location = location;
+    }
+
+    @Override
+    public String toString() {
+        return location;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+            || (other instanceof Name // instanceof handles nulls
```

Comment:
Another instance here. I think should be other instanceof Location right?

Reply:
True